packages/myst-cli/src/process/mdast.ts

```diff
@@ -287,18 +281,18 @@ export async function postProcessMdast(
   const { mdast, dependencies } = mdastPost;
   const fileState = cache.$internalReferences[file];
   const state = pageReferenceStates
-    ? new MultiPageReferenceState(pageReferenceStates, file)
+    ? new MultiPageReferenceResolver(pageReferenceStates, file)
     : fileState;
   // NOTE: This is doing things in place, we should potentially make this a different state?
   const transformers = [
     ...(extraLinkTransformers || []),
     new StaticFileTransformer(session, file), // Links static files and internally linked files
   ];
-  linksTransform(mdast, state.file as VFile, {
+  linksTransform(mdast, state.vfile as VFile, {
```

Comment:
Sometimes (even in this single function) `file` was the filename, sometimes `file` was the `VFile`... now we can just tell by the property name 🧹